.github/workflows/tuist.yml

```diff
@@ -159,11 +159,11 @@ jobs:
         run: sudo xcode-select -switch /Applications/Xcode_$(cat .xcode-version).app
       - name: Run
         run: make workspace/lint
-  
+
   lint-lockfiles:
     name: Lint lockfiles
     runs-on: macos-13
     steps:
       - uses: actions/checkout@v3
       - name: Run
-        run: make workspace/lint/lockfiles
+        run: ./make/tasks/workspace/lint/lockfiles.sh
```

Comment:
Avoid exposing this in tha Makefile to simplify CLI

Reply:
Where do we draw the line between what should be or not in the `Makefile`?